components/ws-manager-bridge/src/prebuild-state-mapper.ts

```diff
@@ -16,74 +18,81 @@ export interface PrebuildUpdate {
 
 @injectable()
 export class PrebuildStateMapper {
-    mapWorkspaceStatusToPrebuild(status: WorkspaceStatus.AsObject): PrebuildUpdate | undefined {
-        if (status.phase === WorkspacePhase.STOPPED) {
-            // Ideally, we'd love to hande STOPPED identical to STOPPING, because we want to assume that all conditions are stable.
-            // For reliabilies sake we don't do it because experiences shows that unstable conditions are one of the most common sources of errors.
+    constructor(@inject(ExperimentsClient) private readonly experimentsClient: ExperimentsClient) {}
```

Comment:
nit: One - or maybe even two! - additional line of spacing would help readability a lot :wink:

Reply:
I thought it was to destined to be this way, but it seems putting a line break after `@inject(ExperimentsClient)` makes prettier happy and leaves the extra lines in there 👍